sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -235,58 +105,6 @@ ConfigurationSetting setting = configurationClient.setConfigurationSetting("some
 ConfigurationSetting retrievedSetting = configurationClient.getConfigurationSetting("some_key", "some_label");
 ```
 
-For conditional request, if you want to conditionally fetch a configuration setting, set `ifChanged` to true. 
```

Comment:
seems a lot of sections been deleted like **create resource**, **authentication**, **examples** and **Trouble Shooting**?

Reply:
Yes, in my opinion we should defer to other documentation when it relates to calling CLI commands, point to a centralised wiki document (which @g2vinay is working on) for authentication, link to `/src/samples/readme.md` for a full list of samples, and refer people to the wiki for troubleshooting.